# DetectedProduct message
# This structure contains the information of a detected product.

# Product type
string type

# Whether or not the product is faulty
bool is_faulty

# Target pose
geometry_msgs/Pose pose
